hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -119,6 +137,7 @@ objects:
         resources:
         - infrastructures
         - apiservers
+        - dnses
```

Comment:
Is this required?

Reply:
That part is, yes.  I added that after getting a permission error.